docs/source/reference/summarization.rst

```diff
@@ -0,0 +1,185 @@
+This task is a subset of Sequence to Sequence tasks, which requires the model to generate a variable length sequence given an input sequence. In our case the article would be our input sequence, and the short description/sentence would be the output sequence from the model.
+
+-----
+
+*********
+Inference
+*********
+
+The :class:`~flash.text.SummarizationTask` is already pre-trained on [XSUM](https://arxiv.org/abs/1808.08745), a dataset of online British Broadcasting Corporation articles.
+
+Use the :class:`~flash.text.SummarizationTask` pretrained model for inference on any string sequence using :func:`~flash.text.SummarizationTask.predict`:
+
+.. code-block:: python
+
+	# import our libraries
+	from flash.text import SummarizationTask
+
+
+    # 2. Load the model from a checkpoint
+    model = SummarizationTask.load_from_checkpoint("https://flash-weights.s3.amazonaws.com/summarization_model_xsum.pt")
+
+    # 2. Perform inference from a sequence
+    predictions = model.predict([
+        """
+        Camilla bought a box of mangoes with a Brixton Â£10 note, introduced last year to try to keep the money of local
+        people within the community.The couple were surrounded by shoppers as they walked along Electric Avenue.
+        They came to Brixton to see work which has started to revitalise the borough.
+        It was Charles' first visit to the area since 1996, when he was accompanied by the former
+        South African president Nelson Mandela.Greengrocer Derek Chong, who has run a stall on Electric Avenue
+        for 20 years, said Camilla had been ""nice and pleasant"" when she purchased the fruit.
+        ""She asked me what was nice, what would I recommend, and I said we've got some nice mangoes.
+        She asked me were they ripe and I said yes - they're from the Dominican Republic.""
+        Mr Chong is one of 170 local retailers who accept the Brixton Pound.
+        Customers exchange traditional pound coins for Brixton Pounds and then spend them at the market
+        or in participating shops.
+        During the visit, Prince Charles spent time talking to youth worker Marcus West, who works with children
+        nearby on an estate off Coldharbour Lane. Mr West said:
+        ""He's on the level, really down-to-earth. They were very cheery. The prince is a lovely man.""
+        He added: ""I told him I was working with young kids and he said, 'Keep up all the good work.'""
+        Prince Charles also visited the Railway Hotel, at the invitation of his charity The Prince's Regeneration Trust.
+        The trust hopes to restore and refurbish the building,
+        where once Jimi Hendrix and The Clash played, as a new community and business centre."
+        """
+    ])
+    print(predictions)
+
+Or on a given dataset:
+
+.. code-block:: python
+
+	# import our libraries
+	from pytorch_lightning import Trainer
+	from flash import download_data
+	from flash.text import SummarizationData, SummarizationTask
```

Comment:
formatting

Reply:
What should the format be? Again copied from the `text_classification.rst`

Reply:
see the alignment, it seems to. be that som section is tab (imports) and set of the code is 4 spaces